wiki/Tournaments/CWC/1/en.md

```diff
@@ -157,91 +157,322 @@ This competition has come to an end and resulted in the following podium:
+| CFC | 1 | **3** | **China** | [#1](https://osu.ppy.sh/community/matches/2354049) |
+| **Chile** | **3** | 0 | State(s) of the Art | *win by default* |
+
+### Group Stage week 5
+
+Saturday, 3 December 2011:
+
+| Group | Team 1 |  |  | Team 2 | Match link |
+| --: | --: | :-: | :-: | :-- | :-- |
+| A | **Viciados em Crack** | **2** | 0 | Indonesia | [#1](https://osu.ppy.sh/community/matches/2114282) |
+| B | **CFC** | **2** | 0 | Old School | *win by default* |
+| D | **French fruit overdose club** | **2** | 0 | Korea | *win by default* |
+
+Sunday, 4 December 2011:
 
-### General rules
+| Group | Team 1 |  |  | Team 2 | Match link |
+| --: | --: | :-: | :-: | :-- | :-- |
+| C | **Team Nub** | **2** | 0 | Indonesia II | [#1](https://osu.ppy.sh/community/matches/2118576) |
+| B | **State(s) of the Art** | **2** | 1 | CFC | [#1](https://osu.ppy.sh/community/matches/2117966) |
+| C | **Red Fury** | **2** | 0 | German Hentai's | [#1](https://osu.ppy.sh/community/matches/2124209) |
+| B | **#filipino** | **2** | 0 | Old School | *win by default* |
+| D | **Catch The Hungarians** | **2** | 0 | Korea | *win by default* |
 
-1. It is a 3 vs 3 Team tournament.
-2. Team size is 3 member and 1 backup member in case someone from the team is missing (if the teams want one).
-3. Teams should be builded with people from the same country. When people missing, the team can team up with another country from the same continent (there are exceptions).
-4. Minimum team number is 16, maximum 32.
-5. No cheating.
-6. If anyone wants to join the staff write me a forum PM.
-7. Maps can be choosed like in Weezy's tournament. Maps will be given on Monday.
+Monday, 5 December 2011:
 
-### Game specific rules
+| Group | Team 1 |  |  | Team 2 | Match link |
+| --: | --: | :-: | :-: | :-- | :-- |
+| B | **State(s) of the Art** | **2** | 1 | Hungary CTB | [#1](https://osu.ppy.sh/community/matches/2123665) |
+| D | Chile | *N/A* | *N/A* | China | *not played¹* |
 
-1. A game will be a 3 vs 3 match.
-2. Games will be hold on weekends.
-3. When two of a team are missing the game will still hold (rematches in special cases or when both teams accept it).
-4. When participants know that they cannot attend to the game, please tell it the teamleader or the staff.
-5. Participants will play without mods (NoVideo is allowed).
-6. The maps will be only ranked/approved (there are exceptions!).
-7. A staff member or me will create the game. When nobody of the staff is online, participants have to open the game by themselves.
-8. The leader of a team makes a screen and gives it to a staff member or me. Same with the multiplayerpage.
+¹ *As per the Tournament Management's decision ([\[1\]](https://osu.ppy.sh/community/forums/posts/1184327) [\[2\]](https://osu.ppy.sh/community/forums/posts/1206142) [\[3\]](https://osu.ppy.sh/community/forums/posts/1187916)), the Chile vs. China match was not played.*
```

Comment:
this should be a hatnote or some other kind of notice

Reply:
there's already a marker here tho :(

![image](https://user-images.githubusercontent.com/36564236/106346257-44bb6a80-62e8-11eb-98e1-6dc1109d96a8.png)

Reply:
iirc italicizing entire single lines is... not preferable

Reply:
hmm, i think this should be okay tho since the sentence is meant to be treated as an annotation/additional note (instead of as text of its own) as follows :

![image](https://user-images.githubusercontent.com/36564236/106355485-d77cf900-632a-11eb-9160-448b93742995.png)

![image](https://user-images.githubusercontent.com/36564236/106355499-e4015180-632a-11eb-8fd0-d7093212b409.png)

Reply:
just checked this and realized i missed the little superscript 1 in the original, mb